astro-iris/prompts.yml

```diff
@@ -11,6 +11,10 @@ repo_name:
     Please enter a directory name for your new project repository.
     Alphanumeric characters, hyphens and underscores are allowed.
     Lowercase is recommended.
+  regexp_validator: "^\\w+(-*\\w+)*$"
+  error_msg: |
```

Comment:
When does this error message appear? Is all validation done at once, then spits out all the error, or it checks each input in turn, then immediately spit out error if one is incorrect?

Reply:
It checks each input in turn and if it is incorrect it emits an error.

Reply:
So, the user will be prompted until he/she enters the correct value (as we had before) :)